test/new-e2e/utils/e2e/client/agent.go

```diff
@@ -30,6 +35,56 @@ func (agent *Agent) init(auth *Authentification, data *agent.ClientData) error {
 	return err
 }
 
-func (agent *Agent) Status() (string, error) {
-	return agent.sshClient.Execute("sudo datadog-agent status")
+func (agent *Agent) Version() (string, error) {
+	return agent.sshClient.Execute("datadog-agent version")
+}
+
+type Status struct {
+	rawString string
+}
+
+func NewStatus(s string) *Status {
+	return &Status{rawString: s}
+}
+
+func (agent *Agent) Status() (*Status, error) {
+	s, err := agent.sshClient.Execute("sudo datadog-agent status")
```

Comment:
Note (Nothing to do): This doesn't work on Linux systems that don't use `sudo`.

Reply:
We need to address commands by os family, this client doesn't work on Windows. Let's keep it as is for now, added AETT-142 to work on it